LayoutTests/fast/forms/onchange-setvalueforuser.html

```diff
@@ -19,13 +19,6 @@
             testRunner.setValueForUser(tf, 'Hello!');
         }
 
-        // Should not fire the event until focus is lost.
```

Comment:
The `change` event typically fires after `focus` is lost for text input types.

See https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event.

Reply:
But we are just autofilling, so the element is never going to get focused in the first place. Does that really make sense here?

Reply:
Well it happens in this test because the test manually does a focus/unfocus before calling setValueForUser, but Epiphany (and presumably Safari) does not do that.

Reply:
Makes sense. This scenario also fits the description of "When the user commits the change explicitly...", in which case it's okay to fire a change event immediately.

How would you feel about updating `LayoutTests/fast/events/onchange-js.html` to assert that a change event is not fired before losing focus when typing – so that we don't lose the coverage that this test provided.

Reply:
Sure. I've attempted this. Let's see if the EWS likes it.